tools/building/updateSourceFiles.py

```diff
@@ -30,6 +30,7 @@ def file_extension(name):
 
 
 def get_cmake_file_paths(root):
+    # TODO: Needs an update?
```

Comment:
TODO

Reply:
@fsimonis Do we need an update in this file to also add new files in `precice/tests`?